src/components/calendar/Calendar.css

```diff
@@ -0,0 +1,129 @@
+.calendar-container {
```

Comment:
Just for the future, we are going to have each css item in this format: filename-classname/id.
Ex: calendar-container, calendar-row-0.. etc.
This will help resolve any css conflicts in the future.